dist/assets/stylesheets/main.css

```diff
@@ -0,0 +1,141 @@
+.sidenav {
+  height: 100%; /* Full-height: remove this if you want "auto" height */
+  width: 15%; /* Set the width of the sidebar */
+  position: fixed; /* Fixed Sidebar (stay in place on scroll) */
+  z-index: 1; /* Stay on top */
+  top: 0; /* Stay at the top */
+  left: 0;
+   background-color: #111;  
+  overflow-x: hidden; /* Disable horizontal scroll */
+  padding-top: 20px;
+}
+.sidenav-content{
+  margin-top: 200px;
+}
+.dropbtn {
+  background-color: #131413;
+  color: white;
+  padding: 16px;
+  font-size: 16px;
+  border: none;
+  width: 210%;
+}
+#project-form{
+margin-top: 30px;
+}
+.projects
+{
+  cursor: pointer;
+}
+#description{
+  width: 79%;
+}
+#task-form {
+  background: gray;
+  padding: 30px 0 50px 0px;
+  border-radius: 10px;
+  /* width: 39%; */
+  margin-left: auto;
+  margin-right: auto;
+  box-shadow: 50px  rgba(0,0,0,0.2);
+  z-index: 100;
+  padding: 10px;
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 50px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0,0,0); /* Fallback color */
+  background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+.content{
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 35%;
+  border-radius: 25px;
+}
+.close{
+  color: #aaaaaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;}
+#task-form h1{
+  font-size: 1.5rem;
+}
+#task-form input{
+  width: 80%;
+  /* margin-left: 6%;  */
+ align-self: center;
+}
+table {
+  margin-left: 20%;
+  margin-top: 15px;
+}
+
+.add-task {
+  cursor: pointer;
+  margin-left: 20%;
+  margin-top: 15px;
+}
+/* .todo-form .input-group {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 18px;
+} */
```

Comment:
## Avoid committing and pushing commented out code.
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use `git history` instead. More on this on this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).